CONTRIBUTING.md

```diff
@@ -24,7 +19,8 @@ Making a Contribution
 1. Write a test which shows that the bug was fixed or that the feature works as expected.
 1. Send a pull request (PR) and work with us until it gets merged and published. Contributions may need some modifications, so a few rounds of review and fixing may be necessary. We will include your name in the ChangeLog :)
 1. For quick merging, the contribution should be short, and concentrated on a single feature or topic. The larger the contribution is, the longer it would take to review it and merge it.
-1. Mbed TLS is released under the Apache license, and as such, all the added files should include the Apache license header.
+1. Ensure that all new files include the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) standard license header.
+1. Ensure that each commit has at least one `Signed-off-by:` line, using your real name and email address. The names in the `Signed-off-by:` and `Author:` lines must match. If anyone else contributes to the commit, they must also add their own `Signed-off-by:` line. By adding this line the contributor(s) certify that the contribution is made under the terms of the [Developer Certificate of Origin](dco.txt).
```

Comment:
Additionally to the dco, we could consider mentioning the inbound licences here too. (The dco makes a vague reference to an open source licence, it is not impossible to figure out what it is, but I think we should make the job easier for our contributors.)

Reply:
OK, I'll link the license section of the readme here.